components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/main/java/org/wso2/carbon/identity/notification/sender/tenant/config/NotificationSenderManagementServiceImpl.java

```diff
@@ -109,6 +110,7 @@ public class NotificationSenderManagementServiceImpl implements NotificationSend
 
     private static final Log log = LogFactory.getLog(NotificationSenderManagementServiceImpl.class);
     public static final int MAX_RETRY_COUNT = 60;
+    Map<String, ConfigurationHandler> configurationHandlerMap = new HashMap<>();
```

Comment:
we can keep this map inside the data holder class, instead of keep it in the service.

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/commit/6feda46da33ad906dc43f6c786db85b7f0bce845